scripts/setup-seafile-mysql.py

```diff
@@ -1227,11 +1231,129 @@ def set_file_perm():
 user_manuals_handler = UserManualHandler()
 # Would be created after AbstractDBConfigurator.ask_use_existing_db()
 db_config = None
+need_pause = True
+
+def get_param_val(arg, env, default=None):
+    return arg if arg else os.environ.get(env, default)
+
+def check_params(args):
+    server_name = get_param_val(args.server_name, 'SERVER_NAME', socket.gethostname())
+    ccnet_config.server_name = ccnet_config.validate_server_name(server_name)
+
+    server_ip = get_param_val(args.server_ip, 'SERVER_IP',
+                              socket.gethostbyname(socket.gethostname()))
+    ccnet_config.ip_or_domain = ccnet_config.validate_server_ip(server_ip)
+
+    fileserver_port = get_param_val(args.fileserver_port, 'FILESERVER_PORT', '8082')
+    seafile_config.fileserver_port = Utils.validate_port(fileserver_port)
+
+    seafile_dir = get_param_val(args.seafile_dir, 'SEAFILE_DIR',
+                                os.path.join(env_mgr.top_dir, 'seafile-data'))
+    seafile_config.seafile_dir = seafile_config.validate_seafile_dir(seafile_dir)
+
+    global db_config
+
+    use_existing_db = get_param_val(args.use_existing_db, 'USE_EXISTING_DB', '1')
+    if use_existing_db == '1':
+        db_config = NewDBConfigurator()
+    elif use_existing_db == '2':
+        db_config = ExistingDBConfigurator()
```

Comment:
Here `USE_EXISTING_DB`'s value is 1 or 2, which doesn't feel intuitive. What about:

- `--use-existing-db=1